pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -540,8 +540,10 @@ func (og *operationGenerator) GenerateMountVolumeFunc(
 			}
 		}
 
+		actualStateOfWorld.MarkVolumeAsBeingSetUp(volumeToMount.VolumeName)
 		// Execute mount
 		mountErr := volumeMounter.SetUp(fsGroup)
+		actualStateOfWorld.MarkVolumeAsSetUp(volumeToMount.VolumeName)
```

Comment:
I dont know if "VolumeSetUp" is the right naming here. Because here you call it whether we error or not, so in the error case the volume isnt "SetUp" per-se.

Maybe "VolumeInSetUp" or "VolumeSetUpPending" is more apt?

Reply:
MarkVolumeAsSetUp was not meant to bear the information about the result of the operation, just the fact it finished.  So what about "MarkVolumeAsSetUpFinished"? "SetUpPending" sounds like we're waiting for it to start which is not true.